box.json.dist

```diff
@@ -13,7 +13,8 @@
     "finder": [
         {
             "name": [
-                "*.php"
+                "*.php",
+                "*.php8"
```

Comment:
args :(
https://github.com/symfony/polyfill/pull/287/files

Reply:
This reminds me the old good `.php3` times :smile: